src/Symfony/Component/Console/Tests/Helper/ProgressIndicatorTest.php

```diff
@@ -109,6 +109,42 @@ public function testCustomIndicatorValues()
         );
     }
 
+    public function testCustomFinishedIndicatorValue()
+    {
+        $bar = new ProgressIndicator($output = $this->getOutputStream(), null, 100, ['a', 'b'], '✅');
+
+        $bar->start('Starting...');
+        usleep(101000);
+        $bar->finish('Done');
+
+
+        rewind($output->getStream());
+
+        $this->assertEquals(
```

Comment:
Could you use `assertSame` maybe? `assertEquals` tends to be replaced by it where possible in the codebase. Same in the test case below.